### DISCUSSION TITLE
📺 Join Live Today:  Intro to Message Extensions and making your app intelligent with Microsoft Graph

### DISCUSSION THREAD

**Discussion by kamalshree** [microsoft/hack-together-teams#40] [Announcements]
**Join us live today (Wed, Jun 7) at 4PM GMT/9AM PT in which we will talk about Message Extensions and making your app intelligent with Microsoft Graph**

![#4 - June 7 - Intro to Message Extensions and making your app intelligent with Microsoft Graph](https://github.com/microsoft/hack-together-teams/assets/3199282/40367918-076f-4765-9652-37f3b607a920)

We will begin with @girliemac introducing about Message extension and also will present some hack ideas that use message extensions. Later Sai Deepthi Kovvuru will be discussing about the Teams Apps extensibility across Microsoft 365 🤩

Next, @sebastienlevert and @maisarissi will introduce the Microsoft Graph and demonstrate it

Join us live and bring any and all questions you might have.

👋 See you there! https://aka.ms/hack-together-teams/session4

**Comment by waldekmastykarz** [comment]
This will be an awesome session for you to learn about message extensions and how to use Microsoft Graph to tap into data and insights stored in Microsoft 365. See you there!

**Reply by sebastienlevert** [reply]
Can't wait to talk to you all on the live stream! Now... Will I be wearing the now famous Graph suit 🦒 Who knows...

**Reply by waldekmastykarz** [reply]
🦒
